example/main.tf

```diff
@@ -85,6 +85,38 @@ resource "google_compute_subnetwork" "vpc_subnetwork" {
   ]
 }
 
+resource "google_compute_router" "router" {
+  name    = "${var.router_name}"
+  network = "${google_compute_network.vpc_network.self_link}"
+
+  bgp {
+    asn = 64514
+  }
+}
+
+resource "google_compute_address" "address" {
+  count = 1
+  name  = "${var.address_name}-${count.index}"
+}
+
+resource "google_compute_router_nat" "nat" {
+  name                               = "${var.nat_name}"
+  router                             = "${google_compute_router.router.name}"
+  nat_ip_allocate_option             = "MANUAL_ONLY"
```

Comment:
Comment as to what this means would be great 😄

Reply:
I've changed this to automatic and added a comment in the new version.